### DISCUSSION TITLE
overlay the advertising with darkness?

### DISCUSSION THREAD

**Discussion by The4Apes** [Dreamlinerm/Netflix-Prime-Auto-Skip#73] [Ideas]
Fast forwarding the advertising is very nice, but advertising is always very bright. The change from movie to advertising triggers epileptic. I need an option to cover the advertising with a dark transparent overlay to "dimming the lights".

**Comment by Dreamlinerm** [comment]
Oh, I am sorry. I did not think of epileptics at all, I will try to address this in the next version. What do you think would be best for epileptics, making the hole screen dark or reducing the brightness to like 20% are probably all possible. With reducing the brightness the images would probably still change fast but it would not be like flashes what triggers epiliepsy right?

**Comment by Dreamlinerm** [comment]
Do you think I should make this an Epilepsy option or the default, I think probably an option is best since for other people its nicer not seeing a black screen.

**Comment by Dreamlinerm** [comment]
I probably don't have time this week to work on this, but maybe next week on the weekend I will start.

**Comment by The4Apes** [comment]
I think an option to reduce the brightness up to 100% while advertising is the best solution. Anybody can choose it's personally.

**Reply by Dreamlinerm** [reply]
OK, then I will create an option with a brightness slider.

**Reply by The4Apes** [reply]
I'm very happy, thank you very much.

**Reply by Dreamlinerm** [reply]
Will add this in about two weeks probably

**Comment by Dreamlinerm** [comment]
Added epilepsy option in v.1.0.71. Open an issue if there is any problem with it
